tests/test_aes.py

```diff
@@ -16,3 +20,41 @@ def test_encrypt_decrypt():
     encrypted_msg = encrypt(key_, msg_, 0)
     txt = decrypt(key_, encrypted_msg, 0)
     assert txt == msg_
+
+
+def test_AEAD_good():
+    key = os.urandom(32)
```

Comment:
I don't think its useful in the main conftest.py file, but i moved them to local fixtures in the test file.